OpenOversight/migrations/versions/2023-08-21-0300_a35aa1a114fa_add_create_and_last_update_columns.py

```diff
@@ -0,0 +1,332 @@
+"""Add create and last_update columns
+
+Revision ID: a35aa1a114fa
+Revises: b38c133bed3c
+Create Date: 2023-08-21 03:00:57.468190
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = "a35aa1a114fa"
+down_revision = "b38c133bed3c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("assignments", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "last_updated_at",
+                sa.DateTime(timezone=True),
+                server_default=sa.text("now()"),
+                nullable=True,
+            )
+        )
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        batch_op.create_foreign_key(
+            "fk_assignments_last_updated_by_users",
```

Comment:
LAST THING! We have a lot of scenarios where there weren't names used for FKs and they are being input as `[table]_[column_name]_fkey`. Could you update the foreign key names to that motif?
<img width="368" alt="Screenshot 2023-08-22 at 3 31 50 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/81a93cc6-2189-4f13-9569-38bdf6de5b82">

Reply:
I'm addressing all of the previous migrations in this PR: https://github.com/lucyparsons/OpenOversight/pull/1036

Reply:
Good callout! I've updated the foreign key names